src/index.html

```diff
@@ -34,40 +34,76 @@
       <nav class="nav">
         <ul class="nav__list">
           <li class="nav__item">
-            <a class="nav__link is-active" href="#apple">apple</a>
+            <a
+              class="nav__link is-active"
+              href="#apple"
+            >
+              apple
+            </a>
           </li>
 
           <li class="nav__item">
-            <a class="nav__link" href="#samsung">samsung</a>
+            <a
+              class="nav__link"
+              href="#samsung"
+            >
+              samsung
+            </a>
           </li>
 
           <li class="nav__item">
-            <a class="nav__link" href="#smartphones">smartphones</a>
+            <a
+              class="nav__link"
+              href="#smartphones"
+            >
+              smartphones
+            </a>
           </li>
 
           <li class="nav__item">
-            <a class="nav__link"
-             href="#laptops_computers"
-             data-qa="hover"
+            <a
+              class="nav__link"
+              href="#laptops_computers"
+              data-qa="hover"
             >
-            laptops & computers
+              laptops & computers
             </a>
           </li>
 
           <li class="nav__item">
-            <a class="nav__link" href="#gadgets">gadgets</a>
+            <a
+              class="nav__link"
+              href="#gadgets"
+            >
+              gadgets
+            </a>
           </li>
 
           <li class="nav__item">
-            <a class="nav__link" href="#tablets">tablets</a>
+            <a
+              class="nav__link"
+              href="#tablets"
+            >
+              tablets
+            </a>
           </li>
 
           <li class="nav__item">
-            <a class="nav__link" href="#photo">Photo</a>
+            <a
+              class="nav__link"
+              href="#photo"
+            >
```

Comment:
If the tag has 2 or fewer attributes u shouldn't move attributes to a new line. Fix in all cases